Atomic/atomic.py

```diff
@@ -678,6 +679,24 @@ def get_vulnerable_ids(self):
         except IOError:
             return []
 
+    def get_local_tokens(self):
+        if len(self.local_tokens) < 1:
+            self.local_tokens = self.load_local_tokens()
+        return self.local_tokens
+
+    @staticmethod
+    def load_local_tokens():
+        tokens = {}
+        token_file_name = os.path.expanduser('~/.docker/config.json')
```

Comment:
@runcom Does Skopeo just do this, or is it a little more cleaver?  Docker?  Does it look at SUDO_USER?